elements/lisk-p2p/src/peer_book/peer_book.ts

```diff
@@ -76,6 +80,19 @@ export class PeerBook {
 		return [...this.newPeers, ...this.triedPeers];
 	}
 
+	public get seedPeers(): ReadonlyArray<P2PPeerInfo> {
+		return this._seedPeers;
+	}
+	public get fixedPeers(): ReadonlyArray<P2PPeerInfo> {
+		return this._fixedPeers;
+	}
+	public get whitelistedPeers(): ReadonlyArray<P2PPeerInfo> {
+		return this._whitelistedPeers;
+	}
+	public get bannedIps(): ReadonlyArray<string> {
+		return [...this._blacklistedIPs, ...this._bannedIps];
```

Comment:
maybe better to return as Set? it could be cloned set.
Probably more easy to use as it has `has`